### PR TITLE
feat(tke-upgrate): add tkestack components upgrade script

### PR DESCRIPTION
**Pull request by shamcleren**
add tkestack components upgrade script

### CODE REVIEW

File: cmd/tke-upgrade/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## How To Upgrate TKEStack
+
+### git clone
+```shell script
+git clone http://git.code.oa.com/eetke/upgrade.git
```
Comment (QianChenglong) [inline cmd/tke-upgrade/README.md:5]: modify

File: cmd/tke-upgrade/app/data/oidc_client_secret [hunk 2]
```diff
@@ -0,0 +1 @@
+17LLDZy9tC5ulFEAu2PomaQpIvF
```
Comment (QianChenglong) [inline cmd/tke-upgrade/app/data/oidc_client_secret:1]: ?

File: cmd/tke-upgrade/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+### find installer data in /opt/tke-install/data and copy to tke-upgrade
+```shell script
+cp /opt/tke-installer/data/tke.json cmd/tke-upgrade/app/data/tke.json
+cp /opt/tke-installer/data/oidc_client_secret cmd/tke-upgrade/app/data/oidc_client_secret
```
Comment (QianChenglong) [inline cmd/tke-upgrade/README.md:11]: ?

File: cmd/tke-upgrade/app/data/oidc_client_secret (hunk 2 above)
Comment (shamcleren) [reply cmd/tke-upgrade/app/data/oidc_client_secret:1]: faker key

File: cmd/tke-upgrade/README.md (hunk 3 above)
Comment (shamcleren) [reply cmd/tke-upgrade/README.md:11]: tkeStack components's upgrade must fetch some cluster infomations where is from tke-installer environment

**Pull request merged**